integration-tests/bats/sql-merge.bats

```diff
@@ -974,8 +973,7 @@ SQL
     [[ "$output" =~ "CONSTRAINT \`c\` CHECK ((\`i\` < 10))" ]] || false
     dolt commit -am "changes to main"
 
-    skip_nbf_not_dolt
-    dolt merge other -m "merge other"
+doltdb.IgnoreTableName    dolt merge other -m "merge other"
```

Comment:
Are you mixing newline char types or something? Seems like all of your newlines are rendering poorly in the GitHub diff view.